src/base/util_func.c

```diff
@@ -537,27 +537,85 @@ util_log_write_warnstr (const char *format, ...)
  *
  * argc (in) :
  * argv (in) :
+ * util_name_pos (in) : This is the position information that specifies the tool name in the argv[] array.
+ *                      If the tool does not take a password as an argument, specify 0 or less.
  */
 int
-util_log_write_command (int argc, char *argv[])
+util_log_write_command (int argc, char *argv[], int util_name_pos)
 {
-  int i;
+  int i, long_pwd_arg_len;
   size_t remained_buf_length, str_len;
   char command_buf[UTIL_LOG_MAX_MSG_SIZE];
   char *p;
+  const char *argv_str;
```

Comment:
```suggestion
  const char *argv_str = NULL;
```
util_pwd_1, util_pwd_2와의 통일성을 위해 다음과 같이 수정해도 좋을 거 같습니다.